libkbfs/ops.go

```diff
@@ -1537,3 +1541,17 @@ func RegisterOps(codec kbfscodec.Codec) {
 	codec.RegisterIfaceSliceType(reflect.TypeOf(opsList{}), opsListCode,
 		opPointerizer)
 }
+
+type pathSortedOps []op
```

Comment:
Might be useful to have a comment on this type to explain that it sorts by length and when it's useful. So basically what you put where it's used:

> Make sure the ops are in increasing order by path length, so e.g. file creates come before file modifies.